pallets/staking/src/lib.rs

```diff
@@ -1315,9 +1319,9 @@ decl_storage! {
         /// forcing into account.
         pub IsCurrentSessionFinal get(fn is_current_session_final): bool = false;
 
-        /// The map from (wannabe) validators to the status of compliance.
-        pub PermissionedValidators get(fn permissioned_validators):
-            map hasher(twox_64_concat) T::AccountId => bool;
+        /// Entities that are allowed to run operator/validator nodes.
+        pub PermissionedEntities get(fn permissioned_entities):
```

Comment:
I think we should stick w/ validator for these.

Reply:
In a code sense, `IdentityId` doesn't represent the validator It is only the representation of the company/project/entity. I don't think it is reasonable to keep the name validator for the storage. If we don't like entity as a suffix then we can use another.

Reply:
Maybe `PermissionedIdentity`? On our terminology page we use:
Identity - relates to the on-chain identity - an entity can have multiple identities
Entity - represents a single "real-life" entity, that can have multiple identities.

Reply:
Yeah that works